src/main/resources/db/migration/V4__task.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS task (
```

Comment:
Jeg oppdaterte code-style i familie-settings med sql. Ser ikke ut til at du har lastet inn code-style på nytt etter det, alternativt har du ikke kjørt kommando for formattering.

Reply:
👍  ikke ofte jeg oppdaterer code-style :)